### ISSUE TITLE
unit for hour and travel time

### ISSUE THREAD

**Issue by locdle** [PSU-Capstone-2014-B/routelandia-android#45]

**Comment by locdle** [comment]
backend will send us 24 hours in string and duration is string in format "min:sec", so do we choose the unit for travel in min or sec or min:sec

**Comment by kl0211** [comment]
Perhaps including seconds might be too precise? I can't think of any application that is that fine-grained with time. Perhaps the minute time could be rounded, and if less than 1 minute, it could just display "< 1". I'd love to hear other opinions on this.

**Comment by donahue2** [comment]
Yeah, if the point of the app is computing travel times across highway distances, seconds is way to granular. Google maps keeps it to the minute, and that has always been plenty sufficient and accurate for me.

**Comment by joshproehl** [comment]
Technically what the stats query is currently producing is not minute:second, it's decimal minutes. 

Which, since I'm lazy, is actually easier to just round to the nearest whole minute rather than convert to min:sec.
